scripts/test.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+set -euxo pipefail
+IFS=$'\n\t'
+
+clean_up () {
+  docker-compose -p fcuitests -f scripts/.tests-compose.yaml down -v
+}
+
+compose_exec() {
+  docker-compose -p fcuitests -f scripts/.tests-compose.yaml exec $@
+}
+
+trap clean_up EXIT HUP INT QUIT PIPE TERM 0 20
+
+docker volume create --name=jar-cache
+docker volume create --name=node-modules
+
+docker-compose -p fcuitests -f scripts/.tests-compose.yaml up -d
+
+docker cp project.clj fcuitests_clojure_1:/w
+docker cp src fcuitests_clojure_1:/w
+docker cp webpack.config.js fcuitests_clojure_1:/w
+
+compose_exec clojure lein with-profile deploy npm install
+compose_exec clojure ./node_modules/webpack/bin/webpack.js -p
+compose_exec clojure lein cljsbuild once
+compose_exec clojure lein resource
+
+docker cp scripts/.phantom-run-tests.js fcuitests_clojure_1:/w/run-tests.js
+compose_exec phantomjs phantomjs run-tests.js
+
+clean_up
```

Comment:
I think `clean_up` is getting called twice. Here's my console output at the end of running this script:

```
Removing network fcuitests_default
WARNING: Network fcuitests_default not found.
Volume node-modules is external, skipping
Removing volume fcuitests_work
WARNING: Volume fcuitests_work not found.
```